distribution/file_handler.py

```diff
@@ -21,7 +21,7 @@ class RemoteFileHandler:
                                'tar.gz', 'tar.xz', 'tar.bz2']
 
     @staticmethod
-    def calculate_md5(fpath, chunk_size):
+    def calculate_md5(fpath, chunk_size=10000):
```

Comment:
I vote for sha256 instead of md5 even when it is used just for file integrity check and not security

Reply:
Ok, changed.